src/libraries/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -210,8 +210,6 @@ public void ActivityIdNonHierarchicalOverflow()
 
             Assert.Equal(parentId, activity.ParentId);
 
-            // With probability 1/MaxLong, Activity.Id length may be expectedIdLength + 1
-            Assert.InRange(activity.Id.Length, expectedIdLength, expectedIdLength + 1);
```

Comment:
System.Diagnostics.Tests.ActivityTests.ActivityIdNonHierarchicalOverflow [FAIL]
        Assert.InRange() Failure
        Range:  (55 - 56)
        Actual: 27
        Stack Trace:
          F:\OSS\runtime\src\libraries\System.Diagnostics.DiagnosticSource\tests\ActivityTests.cs(213,0): at System.Diagnostics.Tests.ActivityTests.ActivityIdNonHierarchicalOverflow()

Reply:
the Activity Id is `|b712496e-47b8baaa43288675.`

Reply:
Ah I see now, the expectedIdLength changed because it was computed from testActivity and the id format of testActivity is now W3C. I'd recommend either calling testActivity.SetIdFormat(IdFormat.Hierarchical) or moving the test to run with the config switch enabled.

Reply:
(And to undo deleting this line)

Reply:
I already have the exact same test included with the flag. I can add `testActivity.SetIdFormat(IdFormat.Hierarchical)`  too here but I am not sure if it is going to add any value more than duplication.

Reply:
Ah, I'd also be fine deleting the copy here. In its current state the test isn't accomplishing much.